deployment/common/types/types.go

```diff
@@ -46,6 +46,8 @@ const (
 	BypasserAccessControllerAccount  deployment.ContractType = "BypasserAccessControllerAccount"
 	// Data Streams contracts
 	ChannelConfigStore deployment.ContractType = "ChannelConfigStore"
+	FeeManager         deployment.ContractType = "FeeManager"
```

Comment:
Can we move these to a domain speCific local types file? Ex in the draft PR https://github.com/smartcontractkit/chainlink/blob/9a3f44a7da8c8aedcb9e6fc6444e7251e314cc04/deployment/data-streams/changeset/types/types.go

or is there a reason to make these global?